poly/main.go

```diff
@@ -111,12 +101,7 @@ func application() *cli.App {
 				Usage:   "Hash a sequence while accounting for circularity using SeqhashV1.",
 
 				Flags: []cli.Flag{
-
-					&cli.StringFlag{
-						Name:  "i",
-						Value: "json",
```

Comment:
Removed this default value to avoid triggering an error each time `hash` was called without a pipe.